src/validators/EmailValidator.test.js

```diff
@@ -0,0 +1,19 @@
+import emailFormatValidator from './EmailValidator.js'
+
+describe('EmailFormatValidator', () => {
+  it('validates 5 digit zip codes', () => {
+    // Invalid Emails
+    expect(emailFormatValidator('test_string').length === 0).toBeFalsy()
+    expect(emailFormatValidator('james@ethoslife.').length === 0).toBeFalsy()
```

Comment:
I think the current email validator allows james@ethoslife.z to be truthy (one character after dot). We should add a spec to confirm that still works. Probably same for `z@ethoslife.com`

Reply:
I'll find the documentation, but the regex I used allows a minimum of 2 characters after the `.`, so  `james@ethoslife.c` wouldn't work, but ` james@ethoslife.co` works. I'll add a test in for that too.